sdcm/cluster.py

```diff
@@ -417,6 +417,33 @@ def wait_cs_installed(self, verbose=True):
         wait.wait_for(func=self.cs_installed, step=60,
                       text=text)
 
+    def update_db_binary(self, new_scylla_bin=None):
+        if new_scylla_bin:
+            self.log.info('Upgrading a DB binary for Node started')
+
+            self.remoter.send_files(new_scylla_bin, '/tmp/scylla', verbose=True)
+
+            # TODO: We should wait for scylla-server.service and scylla-ami-setup.service
+            # has finished instead of sleep here.
+            # As a workaround, sleep 3 minutes hoping they will finish
+            # With --stop-services option in ami, /usr/lib/scylla/scylla_prepare
+            # will fail the scylla-server.service
+            time.sleep(180)
```

Comment:
@kongove if you are interested in this patch, you can take the patch and try to fix the TODO here. It is the real blocker here.